### ISSUE TITLE
Why can data2vec get SOTA vector representation by using the teacher-student method?

### ISSUE THREAD

**Issue by guotong1988** [facebookresearch/fairseq#4422]
https://github.com/pytorch/fairseq/tree/main/examples/data2vec

WHY the teacher-student framework can let the masked-token-pretraining produce SOTA vectors?

thank you very much!
